PULL_REQUEST_TEMPLATE

```diff
@@ -1,11 +1,9 @@
---- erase from here
-
-**Merge/Pull request title name template, please delete it before saving it**
-
+<!--
+Ensure that the title follows this template (or, just use the Asana task title):
 Task_number :: Project :: Story :: Task_type :: Task_name
+-->
+**:link: Asana: ______**
```

Comment:
I see benefit on this:

1. It gives visual feedback: "you need to put something here"
2. It's easy to change, doble click + `cmd+shift+v` (to avoid GitHub creating a link `[___](https://...)`)

But it's OK if you think is better to remove it.